io_scene_godot/converters/multimesh.py

```diff
@@ -0,0 +1,176 @@
+import math
+import bpy
+import mathutils
+
+from ..structures import (
+    Array, NodeTemplate, InternalResource, fix_matrix, mat4_to_string)
+from .mesh import ArrayMeshResourceExporter
+
+
+def export_multimesh_node(escn_file, export_settings,
+                          obj, parent_gd_node):
+    """Export blender particle to a MultiMeshInstance"""
+    context = bpy.context
+    dg = context.evaluated_depsgraph_get()
+    ob = context.object.evaluated_get(dg)
+
+    multimeshid_active=None
+    for i in ob.particle_systems:
+        ps = i
+        if (ps.settings.instance_collection and
+                ps.settings.instance_collection.all_objects[0]):
+            instance_object = ps.settings.instance_collection.all_objects[0]
+        elif ps.settings.instance_object:
+            instance_object = ps.settings.instance_object
+
+        multimeshnode = NodeTemplate(ps.name, 'MultiMeshInstance', parent_gd_node)
+
+        # Export instance mesh resource first
+        instance_mesh_exporter = ArrayMeshResourceExporter(instance_object)
+
+        mesh_id = instance_mesh_exporter.export_mesh(escn_file, export_settings)
+
+        multimeshExporter = MultiMeshResourceExporter(obj, mesh_id, ps)
+
+        multimeshid = multimeshExporter.export_multimesh(
+            escn_file, export_settings, ps.name)
+        
+        if i==ob.particle_systems.active_index:

```

Comment:
I think you mean `i==ob.particle_systems.active` here?  In this loop, `i` is an instance of type `ParticleSystem`.

Reply:
yeah,i will fix it soon